src/runtime/Makefile

```diff
@@ -226,9 +231,13 @@ DEFBINDMOUNTS := []
 DEFSERVICEOFFLOAD ?= false
 
 # SEV Guest Pre-Attestation
+DEFGUESTPREATTESTATION ?= false
+DEFGUESTPREATTESTATIONPROXY ?= localhost:50051
+DEFGUESTPREATTESTATIONKEYSET ?= KEYSET-1
 DEFGUESTPREATTESTATIONSECRETGUID ?= 0a46e24d-478c-4eb1-8696-113eeec3aa99
```

Comment:
This guid is wrong. I think it must have been an error in 4270. The guid for the `offline_sev_kbc` is `e6f5a162-d67f-4750-a67c-5d065f2a9910`. See https://github.com/confidential-containers/attestation-agent/blob/main/src/kbc_modules/offline_sev_kbc/mod.rs#L17

Reply:
Weird. In my testing the other guid worked. Maybe it was defaulting to the right value or something. I've updated it to the guid you specified.

Reply:
Hmmmm. Older versions of the shim didn't allow you to configure he GUID, so it's possible the argument was just being ignored if you were using a non-upstream 4270.